src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java

```diff
@@ -444,6 +514,13 @@ public void sendToPropertyInspector(@Nonnull String context, @Nonnull String act
         sendPayload(eventJson);
     }
 
+    /**
+     * Changes to a profile on the Stream Deck based on the profile's name
+     * 
+     * @param context The unique ID for the plugin, set as the PluginUUID received during registration.
```

Comment:
I should just handle this part myself, feel free to implement this. The UUID is stored in this class

Reply:
Removed context arg from that method and replaced it with `pluginUUID.toString()`